src/main/resources/static/javascript/mainApp/grouping.controller.js

```diff
@@ -809,12 +810,22 @@
          */
         $scope.displayAddModal = (options) => {
             const uhIdentifiers = [].concat(options.uhIdentifiers); // Allows either string or array to be passed in
+            const membersAttributesResults = options.membersAttributes.results;
 
             $scope.listName = options.listName;
             $scope.isMultiAdd = uhIdentifiers.length > 1;
 
+            // Check if there is a dept account to be added.
+            for (const member of membersAttributesResults) {
+                if (member['uid'] === member['uhUuid']) {
+                    $scope.hasDeptAccount = true;
+                    break;
+                }
+                $scope.hasDeptAccount = false;
```

Comment:
Maybe move this above the for-statement?

Reply:
Right, that's a better idea.